js/symbol/shaping.js

```diff
@@ -65,16 +65,42 @@ const invisible = {
 };
 
 const breakable = {
-    0x20:   true, // space
-    0x26:   true, // ampersand
-    0x2b:   true, // plus sign
-    0x2d:   true, // hyphen-minus
-    0x2f:   true, // solidus
-    0xad:   true, // soft hyphen
-    0xb7:   true, // middle dot
+    0x0020: true, // space
```

Comment:
Fortunately we never need to iterate over this object. We just do lookups into it. That should be the fastest solution 👍